### PR TITLE
[do not merge this yet] Main menu example

### PR DESCRIPTION
**Pull request by hanbyul-here**
Introducing `navMainItem`, `navTitle` to tweak main nav. 
Look at https://github.com/NASA-IMPACT/veda-ui/pull/748 for necessary ui change.

### CODE REVIEW

File: custom-pages/visit/index.mdx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
-menu: Visit
+navMainItem: true
+navTitle: Visit
```
Comment (j08lue) [inline custom-pages/visit/index.mdx:3]:
```suggestion
navTitle: Visit the Center
```

**Review by j08lue — CHANGES_REQUESTED** [review]
According to the [design](https://github.com/NASA-IMPACT/veda-config-eic/issues/8#issuecomment-1802023387), we want to call the page "Visit the Center" for clarification.

**Comment by hanbyul-here** [comment]
I also added google form environment variable to see the full scope of nav, the nav breaks its style between 990 and 1021 px viewport width.
Screen Shot 2023-11-14 at 2 56 45 PM

**Review by j08lue — APPROVED** [review]
This solution has been approved and looks good to me https://github.com/NASA-IMPACT/veda-config-eic/issues/22#issuecomment-1815918754

**Pull request closed without merging**